modules/microshift-storage-volume-cloning.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-volume-cloning_{context}"]
+= About LVM volume cloning
+
+The LVMS supports `PersistentVolumeClaims` (PVC) cloning for LVM thin volumes. A clone is a duplicate of an existing volume at that can be used just like any other volume. When provisioned, an exact duplicate of the original volume is created when the `dataSource` references a source PVC in the same namespace. Once created, a cloned PVC is considered a new object and completely separate from the source PVC. The clone represents the data from the source at the moment in time it was created.
```

Comment:
I believe "at" is a typo here. Probably can also remove "just"

Reply:
Are you not able to also add in-line edits? Curious because I see a lot of comments instead of in-line edits, which are much faster for me to apply. Wondering if two people cannot in-line edit?

Reply:
I usually only use the suggested change feature if I'm confident in the edits, but I wasn't positive in this case. It's usually user preference on using that or not, not everyone does